build/rust/Cargo.lock

```diff
@@ -571,6 +629,21 @@ dependencies = [
  "link-cplusplus",
 ]
 
+[[package]]
```

Comment:
this should not be here

Reply:
should be fixed when reverting https://github.com/brave/brave-core/pull/14399/files#diff-9daa86aa87c15c7a01cfdbb2c154bfad8337c0f34b841fed787cc06549400ed9R1233 back to the previous version. Or maybe something is adding a cxx feature? If so it needs to be disabled